src/api/SessionManager.ts

```diff
@@ -29,6 +29,10 @@ export default class SessionManager {
     else return null
   }
 
+  async endCurrentSession(stopCasting: boolean): Promise<void> {
+    await Native.endCurrentSession(stopCasting)
```

Comment:
For consistency, this should return the result of the promise:

```suggestion
    return Native.endCurrentSession(stopCasting)
```

Btw iOS returns a boolean with the status of ending the session. We could approximate that on Android by resolving to false if the session is null or true otherwise.